modules/hdf5_handler/bes-testsuite/hdf5_handlerTest.at

```diff
@@ -679,3 +739,13 @@ AT_BESCMD_CFDMR_RESPONSE_TEST([d_int64.h5.dmr.bescmd])
 AT_BESCMD_CFDMR_DAP4DATA_RESPONSE_TEST([d_int64.h5.dap.bescmd])
 AT_BESCMD_CFDMR_RESPONSE_TEST([t_scalar_dset.h5.dmr.bescmd])
 AT_BESCMD_CFDMR_DAP4DATA_RESPONSE_TEST([t_scalar_dset.h5.dap.bescmd])
+
+# These six tests use the new macro added at the top if the file. jhrg 3/12/22
+# tests for the H5.EscapeUTF8Attr bes key - using the default value of true first
+AT_BESCMD_BESCONF_RESPONSE_TEST([h5.cf/utf8_attr.h5.das.bescmd], [bes.cfdmr.conf])
+AT_BESCMD_BESCONF_RESPONSE_TEST([h5.cf/utf8_attr.h5.ddx.bescmd], [bes.cfdmr.conf])
+AT_BESCMD_BESCONF_RESPONSE_TEST([h5.cf/utf8_attr.h5.dmr.bescmd], [bes.cfdmr.conf], [xfail])
```

Comment:
Why this is the expected failure? I don't understand.

Reply:
It turns out that there is a second function for the DMR attributes that I did not edit. Fixed now.